cypress/objects/taskPage.js

```diff
@@ -0,0 +1,36 @@
+class Tasks {
+    getTasks() {
+        return cy.get('[href="/tasks"]');
+    }
+    getCreateTaskButton() {
+        return cy.get('.btn');
+    }
+    getFormName() {
+        return cy.get('[id="taskDetailsForm.name"]');
+    }
+    getFieldTextArea() {
+        return cy.get('.field__textarea');
+    }
+    getBillToClient() {
+        return cy.get('.Select-value-label');
+    }
+    getBillToClientLabel(value) {
+        return cy.get(`[aria-label=${value}]`);
+    }
+    getFormRate() {
+        return cy.get('[id="taskDetailsForm.rate"]');
+    }
+    getSubmitButton() {
+        return cy.get('[type="submit"]');
+    }
+    getUrl() {
+        return cy.url();
+    }
+    getSmallFieldText() {
```

Comment:
I spent quite a bit figuring out what "SmallFieldText" is :D Try to name your functions logically according to what they refer to and not the name of the CSS class you're getting. For example, you could change `getSmallFieldText()` into `getTaskNameInputField()` or something like that. Makes it a bit clearer what we're talking about, doesn't it? :)